### DISCUSSION TITLE
Inter-probe distance of NP2 alpha and NP2 design

### DISCUSSION THREAD

**Discussion by masahirotaki** [Coen-Lab/chronic-neuropixels#9] [Q&A]
Hi Pip & Celian,

I have a few questions about the NP2 and NP2 alpha implant parts and would greatly appreciate your input.

1. Firstly, we noticed that recently you have updated the ipt files such that you have the designs for both pre-commercial NP2 alpha and commercial NP2 parts. Can you briefly tell us what are some of main changes in terms of parameters? In particular, we are curious if inter-probe distance was calculated based on the wall thickness + the thickness of each probe type? 

2. How problematic do you think it is if we use NP2 alpha parts for commercial NP2 probes? We have tried the alpha version of the parts with the commercial dummy probes, they seem to work fine (except for the inter-probe distance, which was off...)

3. Lastly, we are trying inter-probe distance of 4.6 (ML 2.3) for bilateral recording, which is going beyond the recommended limit. So far the dummy probe implant looks alright, and we will need to do explant to confirm probe condition. However, as mentioned, the probe coordinate seems to be off. I think it may be due to probes not perfectly aligned (despite being corrected to some extent with the tapes) + commercial NP2 being slightly thicker than pre-commercial NP2? Do you have any recommendations for optimising this kind of bilateral design? 

Thank you so much for reading this post. If it is helpful, I am happy to share my current NP2 alpha design.

**Comment by pipcoen** [comment]
Hey Masa,
Answers below. @cbimbo please add/edit anything I may have missed:

1. The new commercial NP2 files are smaller/more compact than the NP2-Alpha files to account for the smaller size of these probes. However, your point about the difference in inter-probe distance is very interesting, and not one we were aware of. Do you have a measurement of this difference? If so, I could adjust the files accordingly. Unfortunately, inter-probe distance will always have a non-zero variation unfortunately... but it would be good to minimise it as far as possible.

2. There are no major differences beyond size (to account for the different form-factor for the commercial probes), so there is no reason to use the new files if you would prefer to use the ones you already have. However, keep in mind that the probes will not fit as "tightly", and the implant will be heavier than it would be otherwise.

3. I've actually recently made a version of the files that extend up to 7.5mm inter-probe distance, which I've emailed you separately. The only reason we haven't uploaded this new version is because it's completely untested, but as soon as someone prints it an confirms that it works, it will replace the current version of the files.

Which changes have you make to the NP2 alpha design? If you think it would be useful to others, definitely upload it via a pull request in the XtraModifications folder (instructions are in the folder)

**Reply by masahirotaki** [reply]
Thank you so much for the reply! They are really helpful. 

1. For the difference between commercial np2 and np2 alpha, unfortunately we haven't had the chance to accurately measure the thickness of the probe. However according to the np website the silicon spacer of the commercial np2 probe is about 1.28mm, but the actual distance from the shanks to the wall should be smaller than that.

Just to confirm, may I ask if inter-probe distance was purely just the thickness of the wall or it accounts for the probe thickness already? If inter-probe distance just means thickness of the wall, then it makes sense that my design is too thick because I didn't account for the probe thickness (+ tapes that I added to correct the angle). 

2. In that case, np2 alpha maybe good enough. I personally found commercial np2 still fits quite nicely. But I do find the window of the docking module for two probes to go through to be quite narrow (especially when I need to add several layers of tapes to adjust shank angle). 

3. Thank you so much! I will have a look!

As of my design, it is just a np2 alpha design with 4.6mm inter-probe distance + 30 degree angle cut. I can share it once it is optimised (especially inter-probe distance) if people find it useful.

**Comment by pipcoen** [comment]
Hey @masahirotaki, sorry I missed the notification for this. 

1. Yes, to confirm, the inter-prove distance is based only on our estimate with NP2-Alpha. It is purely based on the thickness of the middle section and takes nothing else into account, so things like the tape and difference between probes would cause an increased distance. However, I should emphasise that the inter-probe distance is an estimate, and it's difficult to get a precise number because of the "angled" shanks relative to the base.

Please let us know how you get on! Just to confirm, you are using tape to adjust for the slight angles that are produced as a result of the manufacturing process, or something else?

**Reply by masahirotaki** [reply]
Hi @pipcoen, thank you for the clarification! 

Yes, I am using tapes just to adjust for the angles due to manufacturing process.
